File: How_To_Program_Exercises/Visual_C#_How_To_Program2012_Fifth/Chapter6/SummingIntegers.cs
/********************************************************************
 *					Summing Integers								*
 * Program that uses a for statement to sum a sequence of integers	*
 * C++ How To Program Deitel 8th Edition Exercise 5.5				*
 *******************************************************************/
 
using System;

class SummingIntegers
{
	public static void Main(string[] args)
	{
		int n, cardinal, total = 0;
		
		Console.Write("Enter a series of numbers to add (first should specify number of values to be entered): ");
		string [] input = Console.ReadLine().Split();
		
		cardinal = Convert.ToInt32(input[0]);
		
		for (int i = 1; i <= cardinal; ++i)
		{
			n = Convert.ToInt32(input[i]);
			total += n;
		}
		
		Console.WriteLine("\nTotal: {0}", total);
	}
}

File: How_To_Program_Exercises/C++How_To_Program_8/Chapter5/summingIntegers.cpp
/********************************************************************
 *					Summing Integers								*
 * Program that uses a for statement to sum a sequence of integers	*
 * C++ How To Program Deitel 8th Edition Exercise 5.5				*
 *******************************************************************/
 
#include <iostream>

int main()
{
	int total, input, cardinal;
	total = 0;
	
	std::cout<<"Enter a series of numbers to add (first should specify number of values to be entered): ";
	std::cin>>cardinal;
	
	for (int i = 0; i < cardinal; ++i)
	{
		std::cin>>input;
		total += input;
	}
	
	std::cout << "\nTotal: " << total << std::endl;
	return 0;
}